### DISCUSSION TITLE
如何自定义表达式 或者 公式，并显示在编辑器上

### DISCUSSION THREAD

**Discussion by hyzx86** [baidu/amis#6765] [Q&A]
目前通过 registerFilter 注册自定义函数，
但有没有办法像amis-editor 的公式一样呈现在公式列表中？

![image](https://user-images.githubusercontent.com/15613121/235355686-429e7464-fd7a-4c97-b96c-c9f61a6e4956.png)

![image](https://user-images.githubusercontent.com/15613121/235355653-7473b45e-0d9c-4587-900e-102b31f2b882.png)

**Comment by KillenMr** [comment]
自己fork了一份，实现了下

**Reply by hyzx86** [reply]
如果能把系统注册的那个json 对象 导出来，搞个钩子啥的应该就可以了

**Reply by KillenMr** [reply]
或者提供个属性判断是否要覆盖 functions ，能适用更多的类似场景
![image](https://github.com/baidu/amis/assets/76677526/9267be80-701f-4040-b233-bf6f822f0659)
